mapbox/services/surface.py

```diff
@@ -5,11 +5,15 @@
 
 
 class Surface(Service):
-    """Access to the Surface API."""
+    """Access to the Surface API V4"""
+
+    api_name = 'surface'
+    api_version = 'v4'
 
     @property
     def baseuri(self):
-        return 'https://{0}/v4/surface'.format(self.host)
+        return 'https://{0}/{2}/{1}'.format(
```

Comment:
Again this can be inherited from base service?

Reply:
No, for the same reason as directions.